src/ruuvi_endpoint_3.c

```diff
@@ -78,9 +77,8 @@ static void re_3_encode_data (uint8_t * const buffer,
     if (!isnan (re_3_encode_check_invalid (data->humidity_rh, invalid)))
     {
         //Humidity (one lsb is 0.5%, e.g. 128 is 64%). Round the value
-        buffer[RE_3_OFFSET_HUMIDITY] = (uint8_t) ( ( (data->humidity_rh *
-                                       RE_3_ENCODE_HUMIDITY_CONVERT_RATIO) +
-                                       RE_3_ENCODE_HUMIDITY_CONVERT_OFFSET));
+        buffer[RE_3_OFFSET_HUMIDITY] = (uint8_t) roundf (data->humidity_rh *
```

Comment:
Is omitting the HUMIDITY_OFFSET intentional?

Reply:
> Is omitting the HUMIDITY_OFFSET intentional?

HUMIDITY_OFFSET is 0.5 and the original expression looked like `(uint8_t)(rh * ratio + 0.5)`, so I thought that it's just rounding and replaced it with 'roundf'. Or maybe 'roundf' was missed, so we need to add offset and then round?